### ISSUE TITLE
jpackage seems to create it's own jlink:ed image

### ISSUE THREAD

**Issue by lenborje** [beryx/badass-jlink-plugin#39]
I have been using one JDK to run gradle (and thus to compile and link my application), and then specified the Jpackage EA JDK to the jpackage task.

I discovered that the final application then ran with the Jpackage JDK, not the one I've built and jlink:ed with.

The reason seems to be `jpackage create-image` apparently invokes `jlink` to create its own image, unless the option `--runtime-image` is used.

So, to actually use the image produced by the `jlink` task, I must specify the option `--runtime-image` in the `imageOptions`property, i.e.:

```
jlink {
...
    jpackage {
        jpackageHome = //REPLACE WITH JPACKAGE JDK
        imageOptions     = [ '--runtime-image', "$buildDir/image"]
    }
}

```

I believe the `jpackage` task should always specify the image built by the `jlink` task.

If you agree, I can try to implement this and give you a PR.

**Comment by siordache** [comment]
This seems like the right thing to do. I don't know why I didn't implement it this way.

So, a PR is more than welcome. When setting the argument of `--runtime-image`, don't forget to consider the case when the `targetPlatform` method has been called.

**Comment by lenborje** [comment]
OK, I'll look into it.

**Comment by lenborje** [comment]
Solved in PR #43.

**Comment by siordache** [comment]
Available in release 2.8.0.
